test/e2e/framework/providers/azure/doc.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Do we update the dates when we update these files?

Reply:
usually we do, but is a common thing of copying pasting

Reply:
👍🏾